Explorer/Assets/DCL/SDKComponents/MediaStream/Wrapper/MediaPlayerPluginWrapper.cs

```diff
@@ -11,10 +11,11 @@
 using System.Collections.Generic;
 using UnityEngine;
 using RenderHeads.Media.AVProVideo;
-using DCL.ECSComponents;
 using DCL.FeatureFlags;
 using DCL.SDKComponents.MediaStream.Settings;
 using DCL.Settings;
+using System.Linq;
```

Comment:
Where is LINQ?

Reply:
Silly leftover from other ideas. Removing